docs/out-of-process-rpc.md

```diff
@@ -18,7 +19,7 @@ and can be implemented for example in JavaScript messages or deep-link URLs.
 | Name          | Input parameters   | Return type   | Description   |
 |---------------|--------------------|---------------|---------------|
 | getIdentities | `reason`: String<br>`chainIds`: List{String} | `identities`: List{Identity} | Request available identities for the specified chains. The user can choose which of the matching identities they want to reveal to the client. |
-| signAndPost   | `reason`: String<br>`transaction`: UnsignedTransaction | `transactionId`: string | Request signing and posting a transaction prepared by the client. User is asked to confirm the signing after reviewing the transaction content. In case of successful post to the blockchain, the transaction ID is returned to the client to allow tracking the confirmation status. |
+| signAndPost   | `reason`: String<br>`transaction`: UnsignedTransaction | `transactionId`: String or `null` | Request signing and posting a transaction prepared by the client. User is asked to confirm the signing after reviewing the transaction content. In case of successful post to the blockchain, the transaction ID is returned to the client to allow tracking the confirmation status. When the request is denied, `null` is retruned |
```

Comment:
Okay, null not undefined now. I guess this makes it clearer, as undefined can be a forgotten return value, while null must be explicitly returned

Reply:
Good question. These docs are programming language agnostic so `null` is `nil` in Swift, `nullptr` in C++, `nil` in Golang, `null` or `undefined` in JavaScript. But yeah, `null` is probably the better option in case of JSON.